docs/README.md

```diff
@@ -1,12 +1,14 @@
 # Overview
 
-IoT Identity Service is made up of three services that provide provisioning and cryptographic services:
+Azure IoT Edge brings the analytical power of the cloud closer to devices to drive better business insights and enable offline decision making through the use of [Edge modules](https://docs.microsoft.com/azure/iot-edge/iot-edge-modules). The full IoT Edge runtime, in concert with IoT Hub, [provides numerous capabilities](https://docs.microsoft.com/azure/iot-edge/iot-edge-runtime) to scale out and manage an IoT solution from the cloud.
+
+The IoT Identity Service is one component of IoT Edge and can be used stand-alone. The "Identity Service" is technically made up of three services (purple in the diagram) that provide provisioning and cryptographic services. They provide the foundation for authorized components on the device to communicate with Azure services (i.e. IoT Hub) whether as the [device](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry) or an individual [module](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-module-twins). Containerized modules (i.e. IoT Edge modules) require the full IoT Edge runtime. Non-containerized modules (i.e. host-level modules) minimally need the identity, cert, and key services described below.
```

Comment:
In code, packaging and repo structure, IoT Edge depends on Identity Service. Identity Service is NOT a component of IoT Edge. A rephrase of the description for consideration - Identity Service for edge devices helps manage various types of IoT identities and enables crypto operations needed for various applications. Components within IoT Edge depend on the Identity Service.

Reply:
I've updated the wording and shifted a few things around to indicate the relationship between IoT Edge and the identity service.